pkg/driver/node.go

```diff
@@ -514,6 +516,9 @@ func findNvmeVolume(findName string) (device string, err error) {
 
 // getVolumesLimit returns the limit of volumes that the node supports
 func (d *nodeService) getVolumesLimit() int64 {
+	if d.driverOptions.volumeAttachLimit > 0 {
```

Comment:
I think ` d.driverOptions.volumeAttachLimit >= 0` is better, as 0 attach limit is a valid case that ppl might need, and the default -1 will be the option to enable fallback to instance type